lib/note-actions/index.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import onClickOutside from 'react-onclickoutside';
```

Comment:
We should likely replace usage of `react-onclickoutside` with [`focus-trap-react`](https://github.com/focus-trap/focus-trap-react) to improve accessibility of this UI element through trapping the focus and allowing dismissal via pressing <kbd>Esc</kbd>.

#2707 includes adding `focus-trap-react` and can be used as example usage. The PR has not yet merged as it is awaiting review, so we may need to address any conflicts with added packages eventually.

Reply:
Thanks so much. I've updated this for both the NoteActions component and the NoteInfo component.
I've run into a bit of an issue on the NoteActions component though. In it, we use Clipboard.js to copy text. It seems like this doesn't work inside a FocusTrap. An [issue in their repo here](https://github.com/focus-trap/focus-trap-react/issues/51).

Reply:
Marking this as resolved via #2737.